### PR TITLE
People shouldn't be able to start work on more than 3 issues at a time

### PR DESCRIPTION
**Pull request by owocki**
##### Description
Fixes https://github.com/gitcoinco/web/issues/478 - People shouldn't be able to start work on more than 3 issues at a time

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
interest

##### Testing
tested it

##### Refers/Fixes
https://github.com/gitcoinco/web/issues/478

### CODE REVIEW

File: app/dashboard/views.py [hunk 1]
```diff
@@ -83,14 +83,24 @@ def new_interest(request, bounty_id):
...
+            {'error': 'You must be authenticated via github to use this feature!'},
             status=401)
 
     try:
         bounty = Bounty.objects.get(pk=bounty_id)
     except Bounty.DoesNotExist:
         raise Http404
 
+    num_issues = 3
+    active_bounties = Bounty.objects.filter(idx_status__in=['open', 'started'])
```
Comment (mbeacom) [inline app/dashboard/views.py]: Should we use `current().filter(...)` or `filter(current_bounty=True, ...)` ?

File: app/dashboard/views.py [hunk 2]
```diff
@@ -83,14 +83,24 @@ def new_interest(request, bounty_id):
...
+    num_active = Interest.objects.filter(profile_id=profile_id, bounty__in=active_bounties).count()
+    is_working_on_too_much_stuff = num_active > num_issues
```
Comment (mbeacom) [inline app/dashboard/views.py]: Will we want to check `>=` here?

File: app/dashboard/views.py (hunk 1 above)
Comment (owocki) [reply app/dashboard/views.py]: this is why we code review what i wrote at 10pm with a glass of 🥃 in my hand :P

File: app/dashboard/views.py (hunk 2 above)
Comment (owocki) [reply app/dashboard/views.py]: yes

File: app/dashboard/views.py (hunk 1 above)
Comment (owocki) [reply app/dashboard/views.py]: ffcedbfaa204deb8c256e5a8a09f4d8cbe558981

File: app/dashboard/views.py (hunk 2 above)
Comment (owocki) [reply app/dashboard/views.py]: ffcedbfaa204deb8c256e5a8a09f4d8cbe558981

**Pull request merged**